variantlib/api.py

```diff
@@ -94,3 +96,42 @@ def get_variant_hashes_by_priority(
             yield vdesc.hexdigest
     else:
         yield from []
+
+
+@dataclass
+class VariantValidationResult:
```

Comment:
Please move this under `models` directory. I'd like to keep `api.py` as clean as possible